### PR TITLE
Support InitAccessorDeclaration in UseExpressionBodyForAccessors

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/Analyzers/CSharp/Tests/UseExpressionBody/UseExpressionBodyForAccessorsAnalyzerTests.cs [hunk 1]
```diff
@@ -375,6 +434,32 @@ int Goo
 }", options: UseBlockBodyIncludingPropertiesAndIndexers);
         }
 
+        [WorkItem(20350, "https://github.com/dotnet/roslyn/issues/20350")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExpressionBody)]
+        public async Task TestAccessorListFormatting_FixAll/2()
```
Comment (sharwell) [inline src/Analyzers/CSharp/Tests/UseExpressionBody/UseExpressionBodyForAccessorsAnalyzerTests.cs:439]:
```suggestion
        public async Task TestAccessorListFormatting_FixAll2()
```

**Review by sharwell — APPROVED** [review]

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by Youssef1313** [comment]
@sharwell @CyrusNajmabadi Is this ready to merge?

**Comment by Youssef1313** [comment]
@CyrusNajmabadi Is this ready to merge?

**Comment by sharwell** [comment]
It is ! 😄

**Pull request merged**